include/globals.h

```diff
@@ -496,28 +486,27 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
     // Variant of Spectrum set up for a TTGO using a MAX4466 microphone on pin27
 
     // This project is set up as a 48x16 matrix of 16x16 WS2812B panels such as: https://amzn.to/3ABs5DK
-    // It uses an M5StickCPlus which has a microphone and LCD built in:  https://amzn.to/3CrvCFh
     // It displays a spectrum analyzer and music visualizer
 
     #ifndef PROJECT_NAME
     #define PROJECT_NAME            "TTGO"
     #endif
 
-    #define ENABLE_WIFI             1  // Connect to WiFi
+    #define ENABLE_WIFI             1   // Connect to WiFi
     #define INCOMING_WIFI_ENABLED   1   // Accepting incoming color data and commands
     #define WAIT_FOR_WIFI           0   // Hold in setup until we have WiFi - for strips without effects
     #define TIME_BEFORE_LOCAL       2   // How many seconds before the lamp times out and shows local content
     #define ENABLE_WEBSERVER        1   // Turn on the internal webserver
     #define ENABLE_NTP              1   // Set the clock from the web
-    #define ENABLE_OTA              0  // Accept over the air flash updates
+    #define ENABLE_OTA              0   // Accept over the air flash updates
     #define ENABLE_REMOTE           1   // IR Remote Control
     #define ENABLE_AUDIO            1   // Listen for audio from the microphone and process it
 
     #define DEFAULT_EFFECT_INTERVAL     (60*60*24)
 
     #define MAX_BUFFERS     20
 
-    #define LED_PIN0        21          // Note that TFT board on TFTGO uses ping 19, 18, 5, 16, 23, and 4
+    #define LED_PIN0        21          // Note that TFT board on TFTGO uses pins 19, 18, 5, 16, 23, and 4
     #define NUM_CHANNELS    1
     #define RING_SIZE_0     24
     #define BONUS_PIXELS    0
```

Comment:
Well, in this case it's a comment about the TTGO in the TTGO project define block, so in that context I think this is at least defendable. In any case, I just corrected the "ping" typo here in passing because this type of typo came up in a recent discussion (#432, if I'm not mistaken).

As I just mentioned, I think the whole platformio.ini/globals.h combo needs a rather fundamental overhaul at some point. There are some interesting puzzles to solve there though, like the platformio.ini envs always being an intersection between a device and a project, which is what we want to support but also treat as different entities. How to do that (and restructure things in general) is something people can have very strong and different opinions about, so this requires more thought than I've been able to give it up till now.

Anyway, I digress. For now I'd say: enjoy the typo fix, more improvements will come. :)